### DISCUSSION TITLE
[BUG] Dragend event listeners on document(.body) or window are not called when dragging and scrolling for a while

### DISCUSSION THREAD

**Discussion by epicmau5time** [petyosi/react-virtuoso#905] [Q&A]
**Describe the bug**
When rolling custom drag and drop with no library, after dragging the item to the top or bottom of the page so that the list scrolls to what I would assume to be a new "chunk", none of the listeners fire anymore despite them being on the window or document(.body) and not on the div list element itself.

**Reproduction**
[A stripped down version of the issue in the link provided.](https://codesandbox.io/s/pensive-tesla-obiyje?file=/src/App.js)

**To Reproduce**
1. Drag an item without the page scrolling up or down and see the position coordinates change
3. Again without scrolling the page drop the item and the position now says "item dropped"
4. Repeat step one 1 again but this time drag the item to the top or bottom of the page so that the list scrolls
5. After scrolling for a bit. The drag end event is never called despite the on drag events are still being logged to the position.

**Expected behavior**
The on drop event should still be called since it is defined on the window/document itself and not the item

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: chrome, edge, firefox

**Comment by petyosi** [comment]
I'm not an expert in native d&d matters, so I can't comment on what's the problem with the integration above. Moving this to discussions.

**Comment by epicmau5time** [comment]
From what I can tell, it seems like events such as mouseup, pointerup, and dragend are all removed once the list scrolls for a bit. So I would imagine something is dumping those specific window/document event listeners.
